docs/zh_CN/access-control/acl.md

```diff
@@ -6,55 +6,28 @@
 | -------- | -------------- | ---- | ------------------------------------------------------------ |
 | permit   | enum           | 是   | 规则权限：允许：`allow`；拒绝：`deny`                        |
 | action   | enum           | 否   | 指定动作：发布：`publish`；订阅：`subscribe`；发布/订阅： `pubsub` |
-| topics   | Arrary[String] | 否   | 主题或主题过滤器数组                                         |
+| topics   | Array[String]  | 否   | 主题或主题过滤器数组                                         |
 | username | String         | 否   | 用户名若输入值为 "`#`"，表示所有用户                         |
 | clientid | String         | 否   | 客户端 ID 若输入值为 "`#`"，表示所有客户端                   |
 | and      | Array[Map]     | 否   | 与操作                                                       |
 | or       | Array[Map]     | 否   | 或操作                                                       |
 
 ## 配置实例
 
-在以下配置文件中，我们为不同的用户和主题定义了不同的访问规则，实现了灵活的授权检查机制。
+在以下配置文件中，我们为不同的用户和主题定义了不同的访问规则，实现了灵活的授权检查机制。请以正确的格式将规则写入 `nanomq_acl.conf` 文件，并在 `nanomq.conf` 中 `include`，可参照 [访问控制介绍](./introduction.md)。
```

Comment:
```suggestion
在以下配置文件中，我们为不同的用户和主题定义了不同的访问规则，实现了灵活的授权检查机制。请以正确的格式将规则写入 `nanomq_acl.conf` 文件，并在 `nanomq.conf` 中 `include`，可参照 [访问控制介绍](./introduction.md)。
示例：
`acl = {include "/etc/nanomq_acl.conf"}`
```

Reply:
Thanks for your sugession, I will modify it later.